src/sidebar/components/Annotation/test/AnnotationHeader-test.js

```diff
@@ -251,21 +254,58 @@ describe('AnnotationHeader', () => {
     });
 
     describe('annotation document info', () => {
-      it('should render document info if `showDocumentInfo` is enabled', () => {
-        const wrapper = createAnnotationHeader({ showDocumentInfo: true });
+      beforeEach(() => {
+        fakeDomainAndTitle.returns({
+          titleText: 'This document',
+          titleLink: 'http://www.example.com',
+          domain: 'www.foo.com',
+        });
+      });
+
+      it('should not render document info if on sidebar route', () => {
+        fakeStore.route.returns('sidebar');
+        const wrapper = createAnnotationHeader();
 
         const documentInfo = wrapper.find('AnnotationDocumentInfo');
 
-        assert.isTrue(documentInfo.exists());
+        assert.isFalse(documentInfo.exists());
       });
 
-      it('should not render document info if `showDocumentInfo` is not enabled', () => {
-        const wrapper = createAnnotationHeader({ showDocumentInfo: false });
+      it('should not render document info if document does not have a title', () => {
+        fakeStore.route.returns('notebook');
+        fakeDomainAndTitle.returns({});
+
+        const wrapper = createAnnotationHeader();
 
         const documentInfo = wrapper.find('AnnotationDocumentInfo');
 
         assert.isFalse(documentInfo.exists());
       });
+
+      it('should render document info if document info available and not on sidebar route', () => {
```

Comment:
Currently this test is kind of combining two boolean criteria (route-is-sidebar and does-annotation-have-document-info). Might make sense to combine these two tests into parametrized test where the parameters are:

- The current route
- The annotation
- Whether document info is expected to be shown

Reply:
Done.